app/components/sdg/goals/targets_component.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="targets">
+  <ul class="tabs" id="target_tabs" data-deep-link="true" data-tabs>
+    <li class="tabs-title is-active">
+      <%= link_to SDG::Target.model_name.human(count: :other).upcase_first, "#tab_global_targets" %>
+    </li>
+    <li class="tabs-title">
+      <%= link_to SDG::LocalTarget.model_name.human(count: :other).upcase_first, "#tab_local_targets" %>
+    </li>
+  </ul>
+
+  <div class="tabs-content" data-tabs-content="target_tabs">
+    <% [global_targets, local_targets].each do |targets| %>
+      <div class="tabs-panel <%= active(targets) %>" id="tab_<%= type(targets) %>_targets">
+        <% if targets.blank? %>
+          <div class="callout primary text-center">
+            <%= t("sdg.goals.show.targets.empty_local_targets") %>
```

Comment:
While we sometimes use the `empty_` prefix, I think the `no_` prefix is far more common in the code :thinking:.